data/track_categories.yaml

```diff
@@ -7,6 +7,9 @@ genome_track_categories:
           colour: DARK_GREY
           label: Protein coding genes
           track_id: gene-pc-fwd
+          trigger: [track, gene-pc-fwd]
+          display_order: -110
```

Comment:
Not for right now; but for a near future, when we are asked to enable track reordering. We need to think harder about the display order. The numbers that are currently used do not make any sense on the client. The client can use something predictably simple, like 1,2,3,4,5...; but not something as bizarre as -110, -100, 0, 100, 110, 890, 900, 1001...

Reply:
Yes, I just used the current values in GB code and these are meant to be read/applied by GB backend, not client. For user tracks, we need to implement another ordering system.